.github/actions/xahau-ga-build/action.yml

```diff
@@ -28,6 +28,10 @@ inputs:
     description: 'Cache version for invalidation'
     required: false
     default: '1'
+  cache_enabled:
+    description: 'Whether to use cache'
```

Comment:
gha_cache_enabled / 'Whether to use actions/cache (disable for self-hosted with volume mounts)'

I was just recently working on this and it wasn't immediately apparent what "cache_enabled" was

Reply:
I agree.

Perhaps it would be better to keep this PR focused and make changes related to `cache_enable` and `ccache_enable` in a separate PR.

Reply:
It's something new in this PR, so may as well fix it in this one ?

Reply:
ah yes, will fix